rviz2/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ find_package(Qt5 REQUIRED COMPONENTS Widgets)
 add_executable(${PROJECT_NAME}_app
   src/main.cpp
 )
+target_include_directories(${PROJECT_NAME}_app
+  PUBLIC ${Qt5Widgets_INCLUDE_DIRS}
```

Comment:
This should be provided by the `Qt5::Widgets` target linked below (via target include directories). This builds for me on Linux right now...

I'm still looking into it, but if anyone else has time to try this out on Linux and give their feedback, that'd be nice.

Reply:
@dirk-thomas Are you using Qt from the ubuntu packages?

Reply:
@anhosi I believe he's out of communication for a few weeks (vacation), but I believe he said that he was when I discussed it with him.

Reply:
I tried to reproduce this issue with a fresh minimal installation of Ubuntu 16.04. The build instructions lack a couple of required dependencies:
- libgles2-mesa-dev
- libglu1-mesa-dev
- qtbase5-dev
- libyaml-cpp-dev
- build assimp v3.3.1 from [source](https://github.com/assimp/assimp/archive/v3.3.1.tar.gz), the Ubuntu package is broken.

With these dependencies installed the build succeeds for me without this patch. So I cannot reproduce this issue. I have Qt version 5.5.1 installed.

I will prepare a PR to update the build instructions (at least for Ubuntu).

Reply:
We extended the instructions in #117.